docs/user-guide/Building a Scene/How-to-configure-geospatial-coordinates.md

```diff
@@ -19,10 +19,43 @@ To ensure reliability of converting the local coordinates to geospatial ones (ac
 
 - **Dependencies Installed**: Intel® SceneScape deployed, MQTT client installed, and MQTT access credentials configured.
 - **Access and Permissions**: Appropriate access to edit the scene with the UI and receive MQTT messages on the scene regulated topic.
-- **Scene Preparation**: A scene is created as outlined in the [new scene guide](./How-to-create-new-scene.md):
-  - Scene surface map should be rectangular with edges aligned to the X and Y axes (true by design for flat maps loaded from images, but must be aligned by the user for scenes using 3D models). See the next sections for how to verify this condition in practice.
-  - Scene scale (pixels per meter) is set up properly.
-  - The geospatial coordinates of the four map corners have been measured at the scene surface level. Refer to the [Conventions](#conventions) section for how to determine the scene corners.
+
+## Steps to Leverage Built-In Geospatial Map Creation
+
+1. Before launching SceneScape instance, ensure the API keys are configured for the map provider of preference by following [How to configure geospatial map service API keys](How-to-configure-geospatial-map-service-api-keys.md). If the instance is already running, you only need to stop the current Web UI container and start a new instance of it.
+1. Click on "New Scene" button at the top right corner of the SceneScape Web homepage.
+1. Switch the "Map Type" to "Geospatial Map".
+1. Select the provider for which you have configured the API key.
+1. Type the address of interest in the "Location" field and click "Go".
+1. Use the controls on the map frame to zoom in or out as needed.
+1. Click on "Generate Geospatial Bounds & Snapshot" and the following things will automatically occur:
+   - "Output Geospatial Coordinates" field will be set to "Yes"
+   - "Map Corners" will be autopopulated with the geospatial coordinates of the four corners of the map
+   - "Pixels per meter" field is autopopulated with the scale of the scene
```

Comment:
```suggestion
1. Click "Generate Geospatial Bounds & Snapshot" so that:
   - "Output Geospatial Coordinates" field is set to "Yes".
   - "Map Corners" are autopopulated with the geospatial coordinates of the four corners of the map.
   - "Pixels per meter" field is autopopulated with the scale of the scene.
```